install/kubernetes/helm/istio/values.yaml

```diff
@@ -396,3 +396,12 @@ global:
   # This field is set to false by default, so 'helm template ...'
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
+
+  # Provide options to allow user to configure JWT related fields.
+  # More context here: https://github.com/istio/istio/issues/9520
```

Comment:
Also we don't need to put the github issue and code reference in the comment. Instead we need comment describing each params clearly.

Reply:
Done! I thought it was a good idea to reference all usages to once file, so if you change something, you don't need to change the comments in all files.